### DISCUSSION TITLE
分批预处理，不一次性读取

### DISCUSSION THREAD

**Discussion by laiyoi** [fishaudio/Bert-VITS2#320] [Ideas]
bert_gen.py里一个with open干爆了我的内存和虚拟内存。能不能当数据集很大时分批预处理

**Comment by Stardust-minus** [comment]
意义不大，可以自己写一个，我这边从来没遇到过。另外你内存多少()

**Comment by laiyoi** [comment]
内存16G，数据集的wavs文件夹里算上.wav emo和bert一共不到1G，那个模型也才2G，不知道为什么会卡住(windows的表都不走了)

**Comment by Stardust-minus** [comment]
bert模型的话加载是比较吃内存的，可以尝试开1 worker，另外16G跑训练确实捉襟见肘了
